### PR TITLE
added deviceorientation event to window

### PR DESCRIPTION
**Pull request by estelle**
for
https://developer.mozilla.org/en-US/docs/Web/API/Window/deviceorientation_event
as per
mdn/sprints#1284

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -1286,6 +1286,58 @@
...
+      "deviceorientation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/deviceorientation_event",
+          "description": "<code>deviceorientation</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "7"
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (vinyldarkscratch) [inline api/Window.json]: We can add Chrome Android support for the first version released, as such:

```suggestion
              "version_added": "18"
```

File: api/Window.json [hunk 2]
```diff
@@ -1286,6 +1286,58 @@
...
+            },
+            "safari_ios": {
+              "version_added": "4.2"
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
```
Comment (vinyldarkscratch) [inline api/Window.json]: Looks like this was set as "3.0" in the original compat table.

```suggestion
              "version_added": "3"
```

File: api/Window.json [hunk 3]
```diff
@@ -1286,6 +1286,58 @@
           }
         }
       },
+      "deviceorientation": {
```
Comment (Elchi3) [inline api/Window.json]:
```suggestion
      "deviceorientation_event": {
```

File: api/Window.json [hunk 4]
```diff
@@ -1286,6 +1286,58 @@
...
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "3.6"
```
Comment (vinyldarkscratch) [inline api/Window.json]: On the original compatibility table, Firefox has a note that says "Firefox 3.6, 4, and 5 supported [mozOrientation](https://developer.mozilla.org/en-US/docs/Web/Events/MozOrientation) versus the standard `DeviceOrientation` event.".  I think that we should bump this and `firefox_android` up to `6`, based upon the following message on `mozOrientation`'s page:

> This experimental API was removed in Gecko 6.0 (Firefox 6.0 / Thunderbird 6.0 / SeaMonkey 2.3), when support for the standard DeviceOrientationEvent was implemented. You should use that API instead.

I was thinking that we should also add an entry with these version numbers and `"alternative_name": "mozOrientation"`.

Comment (vinyldarkscratch) [inline api/Window.json]: On the original compatibility table, Firefox has a note that says "Firefox 3.6, 4, and 5 supported [mozOrientation](https://developer.mozilla.org/en-US/docs/Web/Events/MozOrientation) versus the standard `DeviceOrientation` event.".  I think that we should bump this and `firefox_android` up to `6`, based upon the following message on `mozOrientation`'s page:

> This experimental API was removed in Gecko 6.0 (Firefox 6.0 / Thunderbird 6.0 / SeaMonkey 2.3), when support for the standard DeviceOrientationEvent was implemented. You should use that API instead.

I was thinking that we should also add an entry with these version numbers and `"alternative_name": "mozOrientation"`.

**Comment by vinyldarkscratch** [comment]
I made a couple of changes and committed them directly into this PR to replace the note with a data block (code suggestions wouldn't really cover it).  It looks perfect now, got the green light from me!  I'll let @Elchi3 have the final say.  :wink:

**Pull request merged**